### ISSUE TITLE
how create own name

### ISSUE THREAD

**Issue by PRO100Zloy** [kolton/d2bot-with-kolbot#2533]
can you create your own game at sonicbot or is it always done randomly?

**Comment by DarkHorseDre** [comment]
I don't think you can change it via config, so you may ave to override the creategame in D2BotSonic.dbj in the format `createGame: function (name, pass, diff, delay)`
e.g.
ControlAction.createGame(gamename, password, Account.get().difficulty, StarterConfig.CreateGameDelay * 1000);

I'd leave the last 2 arguments as-is (so just changes un/pw)
